test/azure/legacy/specification/packagemodemgmtplane/README.md

```diff
@@ -0,0 +1,19 @@
+# Testing package-mode
+
+### Settings
+
+``` yaml
+input-file: ../../../../../node_modules/@microsoft.azure/autorest.testserver/swagger/paging.json
+output-folder: $(python-sdks-folder)/azure/legacy/Expected/AcceptanceTests/PackageModeMgmtPlane
+namespace: azure.package.mode
+package-name: azure-package-mode
+package-pprint-name: Azure Package Mode
```

Comment:
I think instead of passing `--package-mode` as a command line argument in `tasks.py`, you can just add the line `packge-mode: mgmtplane` in the readme, so it's less confusing when reading the readme, and you can clean up tasks.py